third_party/terraform/resources/resource_google_project_services.go

```diff
@@ -28,6 +28,10 @@ func resourceGoogleProjectServices() *schema.Resource {
 		Importer: &schema.ResourceImporter{
 			State: schema.ImportStatePassthrough,
 		},
+        DeprecationMessage: "google_project_services is deprecated - many users reported " +
+        "issues with dependent services that were not resolvable.  Please use google_project_service " +
+        "or the terraform-google-modules/terraform-google-project-factory project_services module.  " +
```

Comment:
Should this be a full link? Might be hard to find otherwise.

```suggestion
        "or the https://github.com/terraform-google-modules/terraform-google-project-factory/tree/master/modules/project_services module.  " +
```

Reply:
Yeah, good call.  Had to do spaces-to-tabs so I can't commit your suggestion through GitHub, but I did commit it.